### PR TITLE
Continuous client

### PR DESCRIPTION
**Pull request by sakridge**

### CODE REVIEW

File: src/bin/client-demo.rs [hunk 1]
```diff
@@ -134,85 +187,84 @@ fn main() {
...
+                    sample_tx_count(thread_addr, exit, maxes, first_count, v, sample_period);
+                })
+                .unwrap()
         })
         .collect();
+
+    let time = Duration::new(time_sec, 0);
+    let now = Instant::now();
+    while now.elapsed() < time {
+        println!(
```
Comment (garious) [inline src/bin/client-demo.rs]: Can you move everything within the while loop to a separate function?

File: src/bin/client-demo.rs [hunk 2]
```diff
@@ -134,85 +187,84 @@ fn main() {
...
+            "Signing transactions... {} {}",
+            keypair_pairs.len(),
+            keypair_pairs[0].len()
+        );
+        let signing_start = Instant::now();
+        let transactions: Vec<_> = keypair_pairs
+            .par_iter()
+            .map(|chunk| Transaction::new(&chunk[0], chunk[1].pubkey(), 1, last_id))
```
Comment (garious) [inline src/bin/client-demo.rs]: I'm curious, how are you calculating TPS if TX signing is inside the loop?

Comment (sakridge) [reply src/bin/client-demo.rs]: the other thread is sampling it.

**Comment by sakridge** [comment]
I don't believe the work has a specific 0.7 issue other than general TPS monitoring/improvement.

**Pull request merged**